packages/loader/container-loader/src/prefetchDocumentStorageService.ts

```diff
@@ -27,7 +27,15 @@ export class PrefetchDocumentStorageService extends DocumentStorageServiceProxy
         return p;
     }
 
+    /**
+     *
+     * @deprecated - only here for back compat, will be removed after release
+     */
     public async read(blobId: string): Promise<string> {
+        return bufferToBase64(await this.cachedRead(blobId));
```

Comment:
same comment as above. Isn't the cache read synchronous now?

Reply:
im not sure how to do this one, if i make read async , then readBlob will have to be synchronous